.gitignore

```diff
@@ -0,0 +1,14 @@
+.DS_Store
+.idea
+
+.ddev/*
```

Comment:
".DS_Store" is a file for configurations on mac. Would be suggestet as "Unversioned Files". This could be bothering when you work on a a mac.
".idea" is the directory from PHPStorm. If you do not exclude this directory, you have it all the time as "Unversioned Files".
With the ddev, i agree. That is something we could remove because it has a own gitignore file.

Reply:
I know. My comment was only about `.ddev/*`